conn_test.go

```diff
@@ -896,6 +937,14 @@ func (srv *TestServer) process(f *framer) {
 			f.writeHeader(0, opError, head.stream)
 			f.writeInt(0x1001)
 			f.writeString("query killed")
+		case "killaftertimeout":
+			atomic.AddInt64(&srv.nKillReq, 1)
+			f.writeHeader(0, opError, head.stream)
+			f.writeInt(0x1001)
+			f.writeString("query killed")
+			rand.Seed(time.Now().UnixNano())
+			sleepFor := time.Duration(time.Millisecond * time.Duration((rand.Intn(20) + 25)))
```

Comment:
can you run this test with stress to ensure it does not flake https://godoc.org/golang.org/x/tools/cmd/stress

Reply:
Yeah, I dropped that addition and went with the existing `slow` query. There's really no need for this randomized timing out of the query here because it isn't successful in either case so just using `slow` and checking that the attempt timeout is used is fine.

Reply:
Oh, and thanks for pointing me in the direction of the `stress` tool. I still let it run over the whole suite and saw that the date marshaling tests are flaky. After disabling that it ran:
```
226 runs so far, 0 failures
^C

real	5m24.240s
user	10m25.325s
sys	11m45.440s
```

I also let the timeout test run under stress on its own and it's all good.

Reply:
Isn't that weird, I can't reproduce it right now with the stress tool. Here's what I did:
- `go test -tags=unit -c`
- `time stress ./gocql.test -test.cpu=8`

I swear it was flaking on the `TestMarshalDate` where sometimes the result would be off by 1 millisecond.

Reply:
stress is mostly useful for running a single test that can execute pretty quickly, found lots of flakes it with.